sound/soc/sof/sof-priv.h

```diff
@@ -74,6 +74,12 @@ extern int sof_core_debug;
 /* max number of DSP cores */
 #define SOF_MAX_DSP_NUM_CORES 8
 
+enum sof_intel_ipc_version {
```

Comment:
I don't think this can be "Intel IPC" version as the IPC is common.

And I'm not sure it's a good to introduce a concept of "IPC version" that is different from SOF ABI major version.  The SOF ABI really is a version of the IPC, so this is a duplicate definition.